lib/ddtrace/contrib/lograge/instrumentation.rb

```diff
@@ -0,0 +1,38 @@
+module Datadog
+  module Contrib
+    module Lograge
+      # Instrumentation for Lograge
+      module Instrumentation
+        def self.included(base)
+          base.prepend(InstanceMethods)
+        end
+
+        # Instance methods for configuration
+        module InstanceMethods
+          def custom_options(event)
+            original_custom_options = super(event)
+
+            # Retrieves trace information for current thread
+            correlation = Datadog.tracer.active_correlation
```

Comment:
Is `correlation` always non-nil?

Reply:
yes, it wil always return a Datadog::Correlation::Identifier struct. https://github.com/DataDog/dd-trace-rb/blob/e702645889ffdc259392e91f30c046352a310778/spec/ddtrace/tracer_spec.rb#L513-L526